Modules/Core/Transform/include/itkTranslationTransform.hxx

```diff
@@ -46,6 +46,12 @@ template <typename TParametersValueType, unsigned int NDimensions>
 void
 TranslationTransform<TParametersValueType, NDimensions>::SetParameters(const ParametersType & parameters)
 {
+  if (parameters.Size() < SpaceDimension)
```

Comment:
I assume that `parameters.Size() > SpaceDimension` is not a problem?

Reply:
> I assume that parameters.Size() > SpaceDimension is not a problem?

@dzenanz Thanks for asking. Honestly I don't know what to do about `parameters.Size() > SpaceDimension`. It was always accepted silently, it never caused a crash, or "undefined behavior". I just don't know if they might sometimes be used elsewhere.